lib/taskana-core/src/main/java/pro/taskana/impl/WorkbasketServiceImpl.java

```diff
@@ -273,7 +274,7 @@ public void deleteWorkbasketAuthorizationForAccessId(String accessId) {
 
     @Override
     public void checkAuthorization(String workbasketId,
```

Comment:
Maybe this Method should be called "checkPermissions"

Reply:
Agreed with Holger to leave the name checkAuthorization